File: PJS5_POC_Cpp/Sources/Pilot/Bot/BotPilot.cpp
#include "BotPilot.h"

namespace PILOT
{
	BOT_PILOT::BOT_PILOT(ROBOT::ROBOT* pRobot, std::vector<int> vFuelsReserve, std::vector<int> vRepairKitsReserve)
		: IPILOT(pRobot, vFuelsReserve, vRepairKitsReserve)
	{
	}

	BOT_PILOT::~BOT_PILOT()
	{
	}
}

File: PJS5_CSharp_Form/PJS5_CSharp_Form/Sources/Pilot/Bot/BotPilot.cs
using PILOT;
using PJS5_CSharp.Sources.Robot;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PJS5_CSharp.Sources.Pilot.Bot
{
    public class BOT_PILOT : IPILOT
    {
        public BOT_PILOT(Robot.Robot pRobot, List<int> vFuelsReserve, List<int> vRepairKitsReserve) : base(pRobot, vFuelsReserve, vRepairKitsReserve)
        {
        }

        public List<int> GetFuelsReserve()
        {
            throw new NotImplementedException();
        }

        public List<int> GetRepairKitsReserve()
        {
            throw new NotImplementedException();
        }

        override
        public void PlayTurn(Robot.Robot ennemiRobot)
        {
            Console.WriteLine("BotPilot.cs PlayTurn()");
        }


    }
}
